<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ass_gousset</class>
 <widget class="QWidget" name="Ass_gousset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>Désignation de l'assemblage :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="NomAss">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Assemblage</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="6">
    <widget class="QTextEdit" name="textEdit">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>C18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C24</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Classe de bois</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Classe de service</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pondérations</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="comboPonderations">
        <item>
         <property name="text">
          <string>1.35 G + 1.5 S</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Type de pointes</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="Lg_pointes">
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>70</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>110</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="Type_pointes">
        <item>
         <property name="text">
          <string>Lisses</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Annelées</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="D_pointes">
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="D_tete_pointe">
        <item>
         <property name="text">
          <string>6.8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>D tête pointe:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>H=</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_H1"/>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Diagonale</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Effort</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>DaN</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Angle</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QComboBox" name="comptrac1">
        <item>
         <property name="text">
          <string>Comp.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tract.</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_B1"/>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>B=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="Nom_diag_d1">
        <property name="text">
         <string>d1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>H=</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_H2"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Diagonale</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Effort</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>DaN</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Angle</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="comptrac2">
        <item>
         <property name="text">
          <string>Comp.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tract</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_B2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>B=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="Nom_diag_d2">
        <property name="text">
         <string>d2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_5" native="true">
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_Bentrait"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_Hentrait"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>H=</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>B=</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Pièce recevant l'assemblage</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>Barre :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="Nom_barre_entrait"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget_6" native="true">
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="1" column="0">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Epaisseur</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_ep_agglo">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>Gousset</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="Calculer">
     <property name="minimumSize">
      <size>
       <width>197</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Calculer</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="bouton_Imp">
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
